Formula/go.rb

```diff
@@ -79,16 +79,6 @@ def install
     end
   end
 
-  def caveats; <<-EOS.undent
```

Comment:
I think the default GOPATH situation is fine; why would it not be? Similarly, why would I want to add the GOROOT location to my path? I've written Go in anger and this provides no insight as to why I'd do that so seems useless.